docs/tests/manual-scenarios.md

```diff
@@ -0,0 +1,93 @@
+**Nazwa scenariusza**: Logowanie
+
+**ID**: S1
+
+**Opis**: Zweryfikowanie poprawności działania procesu logowania
+
+**Czynności przygotowawcze**: Utworzenie konta o danych `person/apsi`
+
+**Czynności końcowe**: Usunięcie konta `person/apsi` (o ile wcześniej nie istniało)
```

Comment:
czemu o ile wcześniej nie istniało? ten komentarz w nawiasie bardziej pasuje do utworzenia konta. (+ czy tu nie trzeba bardziej szczegółowo napisać w jaki sposób stworzyć i usunąć konto?)

Reply:
> czemu o ile wcześniej nie istniało?
Dlatego, że na potrzeby przypadku testowego zmieniamy stan aplikacji. Po jego zakończeniu, powinien on zostać przywrócony do stanu sprzed testu (jest to odwrotność czynności przygotowawczej). Z racji, że `person/apsi` jest tworzony w tej chwili domyślnie, to znaczy, że nie ma potrzeby jego usuwania.
> czy tu nie trzeba bardziej szczegółowo napisać w jaki sposób stworzyć i usunąć konto?
Celem tego testu nie jest opis tworzenia i usuwania konta. Uznajemy to za wiedzę ogólną, którą tester posiada